src/gui/src/Action.cpp

```diff
@@ -45,14 +46,16 @@ Action::Action() :
 
 QString Action::text() const
 {
-    QString text = QString(m_ActionTypeNames[keySequence().isMouseButton() ? type() + 6 : type()  ]) + "(";
+    /* The server config parser don't support () with no argument inside, but support no () */
```

Comment:
I think this comment should be clarified.

Reply:
done